files/index/staff.js

```diff
@@ -0,0 +1,24 @@
+$(function() {
+    $(".password-page").hide();
+    var vars = {}, hash;
+    var hashes = window.location.href.slice(window.location.href.indexOf('?') + 1).split('&');
+    for(var i = 0; i < hashes.length; i++) {
+        hash = hashes[i].split('=');
+        vars[hash[0]] = hash[1];
+    }
+
+    if (vars["staff"] === "true") {
+        $(".main").hide();
+        $(".password-page").show();
+    }
+
+    $(".password-button").click(function() {
+        if ($(".password-input").val() === "odinisthebest") {
```

Comment:
btw jquery is very cool

Reply:
This is so great, thanks for sorting!! I agree with @EmmaLRussell on the password front - unlikely this will be used in front of participants (except in case of absolute emergency) so removing password sounds good to me :-)